src/Common/TTLCachePolicy.h

```diff
@@ -117,12 +117,14 @@ class TTLCachePolicy : public ICachePolicy<Key, Mapped, HashFunction, WeightFunc
 
     void setMaxCount(size_t max_count_) override
     {
+        LOG_TRACE(getLogger("QueryCache"), "TTLCachePolicy::setMaxCount (max_count: {}, count(): {})", max_count_, count()); /// TODO: Remove once #66887 is fixed
```

Comment:
Why don't you have such logs under `LOG_TEST`?
Seems much safer and we don't have to remove it later on